gb-backend/grails-app/controllers/nl/thehyve/gb/backend/QueryController.groovy

```diff
@@ -80,11 +82,14 @@ class QueryController extends AbstractController {
             response.characterEncoding = 'utf-8'
             BindingHelper.write(response.outputStream, query)
         } catch (InvalidArgumentsException | InvalidRequestException e) {
+            log.error('Failed to save query because of invalid request or its arguments', e)
             handleBadRequestResponse(e)
         } catch (ResourceAccessException e) {
+            log.error('Failed to save query because of problem with resource access', e)
```

Comment:
This sentence could be misinterpreted as 'access denied'. Maybe better to specify that 'the resource service is unavailable' or sth like this?

Reply:
ah, I see. Thanks!